terragrunt/modules/bors/main.tf

```diff
@@ -446,10 +446,13 @@ data "aws_vpc" "default" {
 resource "aws_db_instance" "primary" {
   db_name                 = "bors"
   engine                  = "postgres"
-  engine_version          = "15.5"
+  engine_version          = "15.8"
   instance_class          = "db.t4g.micro"
   backup_retention_period = 7
 
+  # Make the instance accessible from outside the VPC
+  publicly_accessible = true
```

Comment:
without this, connecting from bastion times out.
Connecting from my laptop still times out after making the DB publicly accessible because we have the ingress rule that limits the connection to the bastion ip only 👍